### DISCUSSION TITLE
web3uikit installation error

### DISCUSSION THREAD

**Discussion by hamioo** [WalletConnect/walletconnect-monorepo#3040] [Developer Support] [answered]
Hi

I'm trying to add"web3uikit" by using yarn add web3uikit
but in returns a bunch of warnings and then stops and doesn't add to project.

this is the log :

    yarn add v1.22.19
    [1/4] Resolving packages...
    warning web3uikit > @web3uikit/web3 > @web3auth/web3auth@1.2.4: Package no longer supported. Contact Support at https://www.npmjs.com/support 
    for more info.
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider@1.8.0: WalletConnect's v1 SDKs are now deprecated. Please upgrade to a v2 SDK. For details see: https://docs.walletconnect.com/
    warning web3uikit > @web3uikit/web3 > @web3auth/web3auth > @web3auth/core@1.2.0: Package has been renamed to @web3auth/no-modal
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > @walletconnect/types@1.8.0: WalletConnect's v1 SDKs are now deprecated. Please upgrade to a v2 SDK. For details see: https://docs.walletconnect.com/
    warning web3uikit > @web3uikit/web3 > @web3auth/web3auth > @web3auth/wallet-connect-v1-adapter > @walletconnect/types@1.7.8: WalletConnect's v1 SDKs are now deprecated. Please upgrade to a v2 SDK. For details see: https://docs.walletconnect.com/
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > @walletconnect/utils > @walletconnect/types@1.8.0: WalletConnect's v1 SDKs are now deprecated. Please upgrade to a v2 SDK. For details see: https://docs.walletconnect.com/
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > @walletconnect/http-connection > @walletconnect/types@1.8.0: WalletConnect's v1 SDKs are now deprecated. Please upgrade to a v2 SDK. For details see: https://docs.walletconnect.com/
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > @walletconnect/client@1.8.0: WalletConnect's v1 SDKs are now deprecated. 
    Please upgrade to a v2 SDK. For details see: https://docs.walletconnect.com/
    warning web3uikit > @web3uikit/web3 > @web3auth/web3auth > @web3auth/wallet-connect-v1-adapter > @walletconnect/client@1.7.8: WalletConnect's 
    v1 SDKs are now deprecated. Please upgrade to a v2 SDK. For details see: https://docs.walletconnect.com/
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > @walletconnect/client > @walletconnect/types@1.8.0: WalletConnect's v1 SDKs are now deprecated. Please upgrade to a v2 SDK. For details see: https://docs.walletconnect.com/
    warning web3uikit > @web3uikit/web3 > @web3auth/web3auth > @web3auth/wallet-connect-v1-adapter > @walletconnect/client > @walletconnect/types@1.8.0: WalletConnect's v1 SDKs are now deprecated. Please upgrade to a v2 SDK. For details see: https://docs.walletconnect.com/
    warning web3uikit > @web3uikit/web3 > @web3auth/web3auth > @web3auth/openlogin-adapter > @web3auth/ethereum-provider > @walletconnect/types@1.7.8: WalletConnect's v1 SDKs are now deprecated. Please upgrade to a v2 SDK. For details see: https://docs.walletconnect.com/
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > @walletconnect/qrcode-modal@1.8.0: WalletConnect's v1 SDKs are now deprecated. Please upgrade to a v2 SDK. For details see: https://docs.walletconnect.com/
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > @walletconnect/qrcode-modal > @walletconnect/types@1.8.0: WalletConnect's v1 SDKs are now deprecated. Please upgrade to a v2 SDK. For details see: https://docs.walletconnect.com/
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > web3-provider-engine > eth-sig-util@1.4.2: Deprecated in favor of '@metamask/eth-sig-util'
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > web3-provider-engine > eth-json-rpc-middleware > eth-sig-util@1.4.2: Deprecated in favor of '@metamask/eth-sig-util'
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > web3-provider-engine > ethereumjs-tx@1.3.7: New package name format for new versions: @ethereumjs/tx. Please update.
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > web3-provider-engine > ethereumjs-block@1.7.1: New package name format for new versions: @ethereumjs/block. Please update.
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > web3-provider-engine > ethereumjs-block > ethereumjs-tx@1.3.7: New package name format for new versions: @ethereumjs/tx. Please update.
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > web3-provider-engine > ethereumjs-vm@2.6.0: New package name format for new versions: @ethereumjs/vm. Please update.
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > web3-provider-engine > ethereumjs-vm > ethereumjs-block@2.2.2: New package name format for new versions: @ethereumjs/block. Please update.
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > web3-provider-engine > ethereumjs-vm > ethereumjs-block > ethereumjs-tx@2.1.2: New package name format for new versions: @ethereumjs/tx. Please update.
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > web3-provider-engine > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > @walletconnect/utils > @walletconnect/browser-utils > @walletconnect/types@1.8.0: WalletConnect's v1 SDKs are now deprecated. Please upgrade to a v2 SDK. For details see: https://docs.walletconnect.com/
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > @walletconnect/client > @walletconnect/core > @walletconnect/types@1.8.0: WalletConnect's v1 SDKs are now deprecated. Please upgrade to a v2 SDK. For details see: https://docs.walletconnect.com/
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > @walletconnect/qrcode-modal > @walletconnect/mobile-registry@1.4.0: Deprecated in favor of dynamic registry available from: https://github.com/walletconnect/walletconnect-registry
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > web3-provider-engine > eth-block-tracker > safe-event-emitter@1.0.1: Renamed to @metamask/safe-event-emitter
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > web3-provider-engine > eth-json-rpc-middleware > safe-event-emitter@1.0.1: Renamed to @metamask/safe-event-emitter
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > web3-provider-engine > eth-json-rpc-infura > json-rpc-engine > safe-event-emitter@1.0.1: Renamed to @metamask/safe-event-emitter
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > @walletconnect/client > @walletconnect/iso-crypto > @walletconnect/types@1.8.0: WalletConnect's v1 SDKs are now deprecated. Please upgrade to a v2 SDK. For details see: https://docs.walletconnect.com/
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > web3-provider-engine > ethereumjs-vm > ethereumjs-common@1.5.2: New package name format for new versions: @ethereumjs/common. Please update.
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > web3-provider-engine > ethereumjs-vm > ethereumjs-block > ethereumjs-common@1.5.2: New package name format for new versions: @ethereumjs/common. Please update.
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > web3-provider-engine > ethereumjs-vm > ethereumjs-block > ethereumjs-tx > ethereumjs-common@1.5.2: New package name format for new versions: @ethereumjs/common. Please update.
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > web3-provider-engine > request > har-validator@5.1.5: this library is no 
    longer supported
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > web3-provider-engine > request > uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
    warning web3uikit > @web3uikit/web3 > @walletconnect/web3-provider > @walletconnect/client > @walletconnect/core > @walletconnect/socket-transport > @walletconnect/types@1.8.0: WalletConnect's v1 SDKs are now deprecated. Please upgrade to a v2 SDK. For details see: https://docs.walletconnect.com/
    [2/4] Fetching packages...
    error @toruslabs/openlogin-jrpc@4.7.0: The engine "node" is incompatible with this module. Expected version ">=16.18.1". Got "16.15.1"
    error Found incompatible module.
    info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.

what should I do?
Here is the [repo](https://github.com/hamioo/nextjs-moralis-nft-marketplace)

**Comment by finessevanes** [comment, marked as answer]
Hey there - you may need to contact the repo owners to let them know they'll need to update your packages from v1 to v2 as we don't maintain `web3uikit`
